cmd/distil-classify/main.go

```diff
@@ -84,8 +85,8 @@ func main() {
 		outputFilePath := c.String("output")
 
 		// initialize client
-		log.Infof("Using pipeline runner interface at `%s` ", endpoint)
-		client, err := compute.NewRunner(endpoint, true, "distil-ingest", 60, 10, true)
+		log.Infof("Using TA2 interface at `%s` ", endpoint)
+		client, err := compute.NewClient(endpoint, true, "distil-ingest", "TA2", 5*60*time.Second, 1000, true, nil)
```

Comment:
`5*60time.Second` and `1000` should both be constants shared across the various ingest files.